src/ui/server/shared-validation/feedback/index.ts

```diff
@@ -0,0 +1,26 @@
+import generateValidationErrors from '../../helpers/validation';
+import isAboveMaxLength from '../../helpers/is-above-max-length';
+
+/**
+ * validates feedback character count input
+ * @param {string} fieldBody
+ * @param {string} fieldId
+ * @param {string} errorMessage
+ * @param {object} errors
+ * @param {number} maximum
+ * @returns {object} errors
+ */
+const validateFeedbackInput = (fieldBody: string, fieldId: string, errorMessage: string, errors: object, maximum: number) => {
+  let updatedErrors = errors;
+
+  // is over maximum number of characters then will be false
+  if (isAboveMaxLength(fieldBody, maximum)) {
```

Comment:
Nice, could this shared validation be called `max-length`? Then it can be used for any max length fields instead of being labelled specific for feedback